### DISCUSSION TITLE
参考文献在文章尾部为什么作者名字都是大写？

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#102] [Q&A]
参考文献在文章尾部为什么作者名字都是大写？我试着引用了一下，明明在bib中作者名字不是都是大写的。

**Comment by tanukihee** [comment]
国标规定如此

**Reply by earthmoon** [reply]
而且只有名，没有姓，是这样吗，好奇怪啊

**Reply by sikouhjw** [reply]
> 而且只有名，没有姓，是这样吗，好奇怪啊

麻烦给出最小工作示例，姓名都会著录并且大写。

**Reply by earthmoon** [reply]
是在文章引用的位置，只有名没有姓。在论文结尾是大写，但都有姓名，名是全的，姓是一个字母。

**Reply by sikouhjw** [reply]
麻烦了解一下国标
![图片](https://user-images.githubusercontent.com/49770925/162430386-32b7f68b-0281-414d-aa2f-c51ad4d80e97.png)
![图片](https://user-images.githubusercontent.com/49770925/162430329-c268bedc-f740-4d27-8f5e-83d4438a42fc.png)

**Reply by earthmoon** [reply]
10.2.1第一行说的是，姓和年构成，但是英文名字在模板中是名和年构成。英文的话，名在前。我看文中的引用，只留下前半段，说明留的是名。

**Reply by earthmoon** [reply]
10.2.1中说，若是单纯姓无法识别改人，可采取姓名两者都要。没有说只要名的。

**Reply by tanukihee** [reply]
> 10.2.1第一行说的是，姓和年构成，但是英文名字在模板中是名和年构成。英文的话，名在前。我看文中的引用，只留下前半段，说明留的是名。

国标的西方姓名也是姓前名后的

**Reply by sikouhjw** [reply]
> 英文的话，名在前。

你错了，国标里在前面的是姓，名是被缩写的。

**Reply by earthmoon** [reply]
> > 10.2.1第一行说的是，姓和年构成，但是英文名字在模板中是名和年构成。英文的话，名在前。我看文中的引用，只留下前半段，说明留的是名。
> 
> 国标的西方姓名也是姓前名后的

国标好奇怪，还是我太奇怪了。。。

**Reply by earthmoon** [reply]
> > 英文的话，名在前。
> 
> 你错了，国标里在前面的是姓，名是被缩写的。

为什么不保留姓和名呢，这样似乎既不奇怪也符合国标

**Reply by sikouhjw** [reply]
> 为什么不保留姓和名呢，这样似乎既不奇怪也符合国标

奇不奇怪是个人主观感受，符不符合国标不是你说的算的，国标规定引用的时候只需要姓氏和年份。

**Reply by earthmoon** [reply]
也是

**Reply by zepinglee** [reply]
> 国标好奇怪，还是我太奇怪了。。。

少见多怪，你可以先看一下 APA style https://apastyle.apa.org/style-grammar-guidelines/references/examples/journal-article-references

**Reply by earthmoon** [reply]
嗯嗯

**Comment by tanukihee** [comment]
用 LaTeX 不就是图省心吗……在水平到达一定程度之前，纠结每个方向的细节，只能说不适合用 LaTeX（

**Reply by earthmoon** [reply]
也是吧，latex要是都会不知道要花多少时间了。
